centralized-sampling-tests/integration-tests/src/main/java/com.amazon.tests/SampleRules.java

```diff
@@ -0,0 +1,253 @@
+package com.amazon.tests;
+
+import org.json.simple.JSONObject;
+
+/** File that contains all sample Rules that will be created to be used for testing */
+public class SampleRules {
+  public String name;
+  public String JSON;
+
+  public double ExpectedSampled;
```

Comment:
I would suggest applying a linter to this code :) The variable naming should be consistent, and these fields probably shouldn't be exposed as `public`.